openmetadata-spec/src/main/resources/json/schema/entity/data/table.json

```diff
@@ -159,6 +161,14 @@
           "items": {
             "type": "string"
           }
+        },
+        "referredColumns": {
```

Comment:
@sureshms, we are not capturing the foreign key references today. Adding referredColumns as an optional field.
Each unique column
dateid references date(dateid) should have a single constraint where dateid will be columns and constraint_type FOREIGN_KEY and referredColumns will have  service_name.db_name.schema_name.date.dateid
if you have a better ideas let me know.
cc @open-metadata/ingestion

Reply:
referred col lgtm